README.md

```diff
@@ -2,23 +2,83 @@
-[![Join the community on Spectrum](https://withspectrum.github.io/badge/badge.svg)](https://spectrum.chat/mobx-state-tree)
+[![Join the community on Github Discussions](https://img.shields.io/badge/GitHub-Discussions-lightgrey)](https://github.com/mobxjs/mobx-state-tree/discussions)
+
+## What is mobx-state-tree?
+
+Technically speaking, mobx-state-tree (also known as MST) is a state container system built on [MobX](https://github.com/mobxjs/mobx), a functional reactive state library.
+
+This may not mean much to you, and that’s okay. I’ll explain it like this: **MobX is a state management "engine", and MST makes it feel a lot more like Redux by giving it structure.** MST also offers better performance and much less boilerplate than Redux!
 
-> Mobx and MST are amazing pieces of software, for me it is the missing brick when you build React based apps. Thanks for the great work!
+MobX is [one of the most popular Redux alternatives](https://2019.stateofjs.com/data-layer/mobx/) and MST is used by companies worldwide. MST plays very well with TypeScript, React, and React Native, especially when paired with [mobx-react](https://github.com/mobxjs/mobx/tree/main/packages/mobx-react). It supports multiple stores, async actions and side effects, enables extremely targeted re-renders for React apps, and much more -- all in a package with _zero dependencies_ other than MobX itself.
+
+_Note: you don't need to know how to use MobX in order to use MST._
 
 # Getting started
 
 See the [Getting started](https://mobx-state-tree.js.org/intro/getting-started) tutorial or follow the free [egghead.io course](https://egghead.io/courses/manage-application-state-with-mobx-state-tree).
 
 👉 Official docs can be found at [http://mobx-state-tree.js.org/](http://mobx-state-tree.js.org/)
 
+## Quick Code Example
+
+There's nothing quite like looking at some code to get a feel for a library. Check out this small example of an author and list of tweets by that author.
+
+```js
+import { types } from "mobx-state-tree"
+
+// Define a couple models
+const Author = types.model({
+    id: types.identifier,
+    firstName: types.string,
+    lastName: types.string
+})
+
+const Tweet = types.model({
+    id: types.identifier,
+    author: types.reference(Author),
+    body: types.string,
+    timestamp: types.number
+})
+
+// Define a store just like a model
+const RootStore = types.model({
+    authors: types.array(Author),
+    tweets: types.array(Tweet)
+})
+
+// Instantiate a couple model instances
+const jamon = Author.create({
+    id: "jamon",
+    firstName: "Jamon",
+    lastName: "Holmgren"
+})
+
+const tweet = Tweet.create({
+    id: "1",
+    author: jamon.id,
+    body: "Hello world!",
+    timestamp: Date.now()
+})
+
+// Now instantiate the store!
+const rootStore = RootStore.create({
+    author: jamon,
+    tweets: [tweet]
+})
+
+// Ready to use in a React component or anywhere:
+const MyComponent = (props) => {
```

Comment:
MST is about manage data and reaction for this in our components. Not only replace for classes with construcrors.
this example not explain that because nothing will change if i change `rootStore.author.firstName`.
maybe more MVP example with one simple input for add tweets =)

Reply:
Fair point -- I think that adding the `observer` from mobx-react-lite will work.

Reply:
now in documentation used onSnapshot callback 
![image](https://user-images.githubusercontent.com/3505959/101223757-ce124000-369d-11eb-956d-73fc7e24f53f.png)
if react developers is not primary auditory we can replace it on reaction that change text of dom node by vanillajs.

Can you make codesandbox or another runnable environment for example ?)